eng/build-job.yml

```diff
@@ -160,6 +160,13 @@ jobs:
             ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
               # Arcade uses this SDK instead of trying to restore one.
               DotNetCoreSdkDir: /usr/local/dotnet
+        # Save nuget packages in pipeline for update dotnet/versions
+        # TODO remove task when https://github.com/dotnet/arcade/issues/1818 is fixed
```

Comment:
It's a bit more nuanced than that. We also need to make sure the Tizen folks can trigger their build based on BAR before removing this. I suggest filing an issue in CoreCLR to track it, not try to capture it in TODO comments.